server/src/middleware/error.js

```diff
@@ -1,3 +1,9 @@
+export function asyncHandler(fn) {
+  return (req, res, next) => {
+    fn(req, res, next).catch(next)
+  }
+}
```

Comment:
What does fn actually do?

Reply:
This will catch any error threw by an async express controller, and pass it to the error handling middleware. You can see many use cases in server/src/api/controllers. Basically, this will prevent any unhandled error from crashing the server.